src/components/customSearch/CustomSearchForm.tsx

```diff
@@ -0,0 +1,96 @@
+import { Grid } from '@mui/material'
+import SubmitListener from 'components/shared/AutoSubmit'
+import { FieldArray, Form, Formik } from 'formik'
+import { FC } from 'react'
+import {
+  CustomSearchQuery,
+  SearchParameter,
+  SimpleSearchQuery
+} from 'types/SimpleSearch'
+import { PAGINATION_LIMIT } from 'utils/constants'
+import AddFieldOrGroupButton from './AddFieldOrGroupButton'
+import FieldGroup from './FieldGroup'
+
+const CustomSearchForm: FC<{
+  onChange: (values: CustomSearchQuery) => void
+}> = ({ onChange }) => {
+  const initialSearchParameter: SearchParameter = {
+    fieldName: '',
+    value: '',
+    distance: 0
+  }
+
+  const initialValues: CustomSearchQuery = {
+    $or: [
+      {
+        parameters: [initialSearchParameter]
+      }
+    ],
+    sortBy: 'uid',
+    sortAsc: true,
+    offset: 0,
+    limit: PAGINATION_LIMIT
+  }
```

Comment:
We could define this constant in the `lib/constants` file. Same goes for "initialSearchParameter"

Reply:
There is a bug that needs to be fixed that involves the `initialSearchParameter`.